### ISSUE TITLE
Problem with table of Pandas DataFrame

### ISSUE THREAD

**Issue by DavidKimDY** [spatialaudio/nbsphinx#594]
I got this diffrence while building GitHub.page

Below picture is what I got the table from Jupyter notebook
<img width="903" alt="image" src="https://user-images.githubusercontent.com/50807669/128652449-998a7839-cd73-4eb4-addd-6575ed486994.png">

But when I uploaded that ipynb file with using nbsphinx, It looks below
<img width="796" alt="image" src="https://user-images.githubusercontent.com/50807669/128652500-7847e861-5104-4b89-9e9d-b1b51378df6e.png">

No striped style, font size is bigger.
Do I have to set some configuration to get the same table on the page?

**Comment by mgeier** [comment]
It is supposed to look closer to your first example, out of the box.

How does https://nbsphinx.readthedocs.io/en/0.8.6/code-cells.html#Pandas-Dataframes look in your browser?

**Comment by DavidKimDY** [comment]
<img width="761" alt="image" src="https://user-images.githubusercontent.com/50807669/129022455-b54df3a6-21f5-4164-b9ab-f03386cea3fa.png">

My browser is working ok even when I did `nbconvert --to html` and opened the html file, It was Ok.

**Comment by mgeier** [comment]
So the dataframe in the `nbsphinx` docs looks as you would expect, right?

Clearly, something is going wrong in your original example. Can you provide a minimal reproducible example?

**Comment by DavidKimDY** [comment]
Oh, It is `conf.py` Cuz I set ` "html_minify" : True` in `html_theme_options` 
It turned back what I need

<img width="760" alt="image" src="https://user-images.githubusercontent.com/50807669/129116185-96472a87-0ad5-4f84-8c10-0e1d916ec966.png">

Thank you for that. I found it when I was making an example.
But, Is there any way how to make this table size get small a little bit or font size in table?
